src/OrchardCore.Modules/OrchardCore.Features/Recipes/Executors/FeatureStep.cs

```diff
@@ -10,7 +11,7 @@ namespace OrchardCore.Features.Recipes.Executors
     /// <summary>
     /// This recipe step enables or disables a set of features.
     /// </summary>
-    public class FeatureStep : IRecipeStepHandler
+    public class FeatureStep : IRecipeStepHandler, IDefineOrder
```

Comment:
Maybe `IOrderedRecipeStepHandler`

Reply:
`IDefineOrder` is a generic name that can be used in multiple scenarios as needed. This way we don't have to use it only for recipe steps.

Reply:
Yes, I meant

    public interface IOrderedRecipeStepHandler : IRecipeStepHandler
    {
        int Order { get; }
    }

Then for the handlers we want to order just replace ` : IRecipeStepHandler` by ` : IOrderedRecipeStepHandler`

Otherwise if like e.g. `IDisposable`, maybe `IOrderable`.

Reply:
@jtkech 
I follow you now. But, why do you recommend that over my approach? I feel that my approach is simpler, no need for inheritance and also, the `IRecipeStepHandler` will never need to depend on `IDefineOrder`